lib/api/core/plugins/pluginContext.js

```diff
@@ -123,7 +142,7 @@ class PluginContext {
           create: pluginRepository.create.bind(pluginRepository),
           createOrReplace: pluginRepository.createOrReplace.bind(pluginRepository),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 83 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)